### PR TITLE
win_virtio_driver_update_test.py:optimize changing cdrom process

### PR DESCRIPTION
**Pull request by xiagao**
replace shutdown original guest and  boot up it with new iso  process with changing cdrom way
ID: 1556772
Signed-off-by: Xiaoling Gao <xiagao@redhat.com>

### CODE REVIEW

**Comment by xiagao** [comment]
(1/2) smp_8.1024m.repeat1.pre_test.Host_RHEL.m7.u5.qcow2.virtio_blk.up.virtio_net.Win8.i386.1.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_native: PASS (1784.72 s)
 (2/2) smp_8.1024m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_blk.up.virtio_net.Win8.i386.1.io-github-autotest-qemu.win_virtio_driver_update_test.with_balloon.before_bg_test.downgrade_upgrade: PASS (792.05 s)

**Comment by xiagao** [comment]
@kimi1978wy   @hereischen  could you help review this patch, thanks.

File: qemu/tests/win_virtio_driver_update_test.py [hunk 1]
```diff
@@ -36,14 +47,14 @@ def run(test, params, env):
...
-        new_params["cdrom_virtio"] = cdrom_virtio_downgrade
-        vm.create(params=new_params)
-        vm.verify_alive()
-        single_driver_install.run(test, new_params, env)
+        change_virtio_media(params["cdrom_virtio_downgrade"])
+        single_driver_install.run(test, params, env)
+        error_context.context("Reboot guest after downgrade virtio driver",
+                              logging.info)
+        vm.reboot(method="system_reset")
         error_context.context("Upgrade virtio driver to original",
```
Comment (kimi1978wy) [inline qemu/tests/win_virtio_driver_update_test.py]: I think "shell reboot" is better than "system reset".

Comment (hereischen) [reply qemu/tests/win_virtio_driver_update_test.py]: agree with @kimi1978wy

**Comment by kimi1978wy** [comment]
Acked-by: Yu Wang <wyu@redhat.com>

**Comment by xiagao** [comment]
@luckyh  got two acks, could you pls help review?

File: qemu/tests/win_virtio_driver_update_test.py [hunk 2]
```diff
@@ -23,11 +26,19 @@ def run(test, params, env):
...
     vm.verify_alive()
 
+    def change_virtio_media(cdrom_virtio):
+        """
+        change iso for virtio-win
+        :param cdrom_virtio: iso file
+        """
+        virtio_iso = utils_misc.get_path(data_dir.get_data_dir(),
+                                         cdrom_virtio)
+        logging.info("Iso is: %s" % virtio_iso)
```
Comment (luckyh) [inline qemu/tests/win_virtio_driver_update_test.py]: Should be `logging.info("Changing virtio iso image to '%s'" % virtio_iso)`

Comment (xiagao) [reply qemu/tests/win_virtio_driver_update_test.py]: Modified, pls have a check.
Thanks.

**Pull request merged**